Sources/OTel/OTel.docc/Logging/logging.md

```diff
@@ -0,0 +1,16 @@
+# Logging
+
+@Metadata {
+    @TitleHeading("Signal")
+    @PageImage(purpose: card, source: "logging-card", alt: "An illustration of logs.")
+    @PageColor(green)
+    @Available("Swift OTel", introduced: "1.0")
+}
+
+OpenTelemetry logging implemented on top of Swift Log.
+
+> Warning: Logging support in Swift OTel is a work in progress. 
+
```

Comment:
Maybe let's say that this is about emitting logs to otel collectors, but you can already use normal swift-log with a metadata provider to just include trace ids in there? 🤔

Reply:
Yeah, good point! I can see how it might be misleading otherwise.